hack/rpm-prepare.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "Reading MGC_VERSION, MCG_RELEASE and GIT_COMMIT env, if they are set"
+# Change version as needed. In most cases MGC_RELEASE would not be touched unless
+# we want to do a re-lease of same version as we are not backporting.
+export MGC_VERSION=${MGC_VERSION:=0.0.1}
```

Comment:
if these are unset should this default to something that makes it obviously a non-production version like `dirty-{gitCommitRev}`?

Reply:
I would like that as well but the release pipeline depends on manually updating the release version number.